src/learning-catalogue/model/event.ts

```diff
@@ -1,17 +1,13 @@
-import {IsNotEmpty, IsPositive, MinDate} from 'class-validator'
+import {IsNotEmpty, IsPositive} from 'class-validator'
 
 export class Event {
 	id: string
 
 	@IsNotEmpty({
-		groups: ['all', 'event.all', 'event.date'],
-		message: 'validation.module.event.date.empty',
+		groups: ['all', 'event.all', 'event.dateRanges'],
+		message: 'validation.module.event.dateRanges.empty',
```

Comment:
I know we have a bit of both in the app right now - but id prefer if we were using underscores to delimit our keys